doc/users/prev_whats_new/whats_new_3.3.0.rst

```diff
@@ -304,43 +415,156 @@ clabels has been changed to (``2 + zorder`` passed to `~.axes.Axes.contour` /
 `~.axes.Axes.contourf`).
 
 
+Better control of ``Axes.pie`` normalization
+--------------------------------------------
+
+Previously, `.Axes.pie` would normalize its input *x* if ``sum(x) > 1``, but
+would do nothing if the sum were less than 1. This can be confusing, so an
+explicit keyword argument *normalize* has been added. By default, the old
+behavior is preserved.
+
+By passing *normalize*, one can explicitly control whether any rescaling takes
+place and whether partial pies should be created.
```

Comment:
i think this is an or - either rescale or partial pie? also is the normalize kwarg still ignored for sum(x)>1?

maybe plot/example?
```python
fig, (ax1, ax2, ax3) = plt.subplots(ncols=3)
ax1.pie([.25,  .3, .5] , normalize=False)
ax2.pie([.25, .3, .5], data, normalize=True)
ax3.pie([2.5, 3, 5], data)?
```

Reply:
It depends; `normalize=False` for `sum(x)>1`  raises an error for an overfull pie.